.github/workflows/ci.yml

```diff
@@ -142,14 +142,14 @@ jobs:
           if [[ "${{ matrix.test-path }}" == "tests/distributed/test_against_external_daemon/" ]]; then
             docker build -f Dockerfiles/debianx.Dockerfile -t jinaai/jina:test-daemon .
             docker run --name jinad -v /var/run/docker.sock:/var/run/docker.sock --network=host -d jinaai/jina:test-daemon
-            pytest --suppress-no-test-exit-code --force-flaky --min-passes 1 --max-runs 5 --cov=jina --cov-report=xml --timeout=300 -v -s ${{ matrix.test-path }}
+            pytest --suppress-no-test-exit-code --force-flaky --min-passes 1 --max-runs 5 --cov=jina --cov-report=xml --timeout=180 -v -s ${{ matrix.test-path }}
```

Comment:
do not set to such a low value, I have seen locally tests taking more than 250 seconds

Reply:
In core? or in the hub? It was set to 120 until two weeks ago. I prefer setting it lower again and increase if we see things failing again.

Reply:
in core, see the PR I opened where this changed, I saw locally tests running in 250 seconds. 180 is not realistic

Reply:
But this timeout is per test and not for all 25 tests as in your PR.

Reply:
no I meant for each test

Reply:
The timeout you are referring to is `timeout-minutes: 15`

Reply:
See #2057

Reply:
Yes. There 25 tests pass in 251 seconds. And this is the total time of all tests. But the 180s is the time per test.

Reply:
ah okey